### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by yzlucas**

### CODE REVIEW

File: client/wfprev/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (ssylver93) [inline client/wfprev/.vscode/extensions.json:1]: .vscode files should be gitignored like in WFNEWS

File: client/wfprev/src/assets/.gitkeep
Comment (ssylver93) [inline client/wfprev/src/assets/.gitkeep:1]: Empty file should be deleted I feel?

File: client/wfprev/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ssylver93) [inline client/wfprev/.vscode/launch.json:1]: Same for this file

File: client/wfprev/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (ssylver93) [inline client/wfprev/.vscode/tasks.json:1]: Same for this file as well

**Review by ssylver93 — CHANGES_REQUESTED** [review]
Just a couple of comments on files I think shouldn't be included, looks good otherwise

**Comment by yzlucas** [comment]
@ssylver93  thanks, modify the gitingore file to get rid of that folder
@sukhpalp  I have updated to use angular 18 now, the duplicate "db" was an accident, I used it for local test only and I have removed the previous one. The 01_00_00/01 create table thing is for testing the liquibase changes success or not, I think we can modify the changelog.json to not excute it if needed.

File: db/scripts/01_00_00/01/ddl/tables/APP_WFPREV.EXAMPLE_CODE.sql
[the remark this answers is not in the document]
Comment (yzlucas) [reply db/scripts/01_00_00/01/ddl/tables/APP_WFPREV.EXAMPLE_CODE.sql:1]: Not necessary, I can keep it on my local. I have removed it and update the changelog.json

**Pull request merged**